modules/administration/pages/ssl-certs-hsts.adoc

```diff
@@ -5,15 +5,21 @@ HTTP Strict Transport Security (https://developer.mozilla.org/en-US/docs/Web/HTT
 
 {productname} allows enabling HSTS, to enable it for a {productname} Server:
 
+.Procedure
 1. Edit `/etc/apache2/conf.d/zz-spacewalk-www.conf`
 2. Uncomment the line `# Header always set Strict-Transport-Security "max-age=63072000; includeSubDomains"`
 3. Restart Apache with `systemctl restart apache2`
 
 To enable it for {productname} Proxies:
 
+.Procedure
 1. Edit `/etc/apache2/conf.d/spacewalk-proxy.conf`
 2. Uncomment the line `# Header always set Strict-Transport-Security "max-age=63072000; includeSubDomains"`
 3. Restart Apache with `systemctl restart apache2`
 
-**IMPORTANT:** Once HSTS is enabled while using the default SSL certificate generated by {productname} or a self-signed certificate, browsers will refuse to connect using HTTPS unless the CA used to sign such certificates is trusted by the browser.
+
+[IMPORTANT] 
+====
+Once HSTS is enabled while using the default SSL certificate generated by {productname} or a self-signed certificate, browsers will refuse to connect using HTTPS unless the CA used to sign such certificates is trusted by the browser.
```

Comment:
Trying to avoid too many "use" words in one sentence ;)
```suggestion
When HSTS is enabled while using the default SSL certificate generated by {productname} or a self-signed certificate, browsers will refuse to connect with HTTPS unless the CA used to sign such certificates is trusted by the browser.
```